server/src/fat/java/com/ibm/ws/lars/rest/ApiTest.java

```diff
@@ -1202,6 +1206,81 @@ public void test500Mapping() throws InvalidJsonAssetException, ParseException, I
         mongoClient.close();
     }
 
+    @Test
+    public void testRuntimeException() throws ClientProtocolException, IOException {
```

Comment:
Might be nice to have a comment in here to say where we are expecting the error to come from, i.e. this test is testing the error servlet, but the one below is testing the exception mapper in jax-rs..? I think ..??